weed/command/filer.go

```diff
@@ -93,6 +93,7 @@ func init() {
 	filerS3Options.port = cmdFiler.Flag.Int("s3.port", 8333, "s3 server http listen port")
 	filerS3Options.portGrpc = cmdFiler.Flag.Int("s3.port.grpc", 0, "s3 server grpc listen port")
 	filerS3Options.domainName = cmdFiler.Flag.String("s3.domainName", "", "suffix of the host name in comma separated list, {bucket}.{domainName}")
+	filerS3Options.dataCenter = cmdFiler.Flag.String("s3.dataCenter", *f.dataCenter, "prefer to read and write to volumes in this data center")
```

Comment:
The default would not work. Need to set this similar to server.go

Reply:
ok